utils/_context/containers.py

```diff
@@ -918,21 +918,22 @@ def __init__(self, host_log_folder) -> None:
         )
 
 
-class SqlServerContainer(SqlDbTestedContainer):
+class MsSqlServerContainer(SqlDbTestedContainer):
     def __init__(self, host_log_folder) -> None:
         self.data_mssql = f"./{host_log_folder}/data-mssql"
         healthcheck = {}
         if not platform.processor().startswith("arm"):
             # [!NOTE] sqlcmd tool is not available inside the ARM64 version of SQL Edge containers.
             # see https://hub.docker.com/_/microsoft-azure-sql-edge
```

Comment:
I think with the new image we should remove this ?

Reply:
I didn't had the time to check, so I kept it in case. If you're able to verify that it's ok, you can remove it

Reply:
I created https://github.com/DataDog/system-tests/pull/3108 as a follow up